### PR TITLE
Detail card

### PR DESCRIPTION
**Pull request by filwie**
This PR implements #25 

What is added:
* small refactor of scss files
* node detail card

How it looks:
orca-ui-card

### CODE REVIEW

File: src/components/NodeDetailCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  render() {
+    return (
+      <div className={`card node-info-card ${this.state.hidden ? 'hidden' : ''}`}>
+        <button type="button" className="close" aria-label="Close" onClick={this.hide}>
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <div className="card-body">
+          <h5 className="card-title">{this.state.nodeData.kind.replace('_', ' ')}</h5>
+          <div className="card-text node-info-text">
+            {/* <pre>{JSON.stringify(this.state.nodeData, null, 2)}</pre> */}
```
Comment (k-jano) [inline src/components/NodeDetailCard.js:39]: Please get rid of unnecessary comments

File: src/components/NodeDetailCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <pre>{JSON.stringify(this.state.nodeData, null, 2)}</pre>
+            {/* dupa */}
```
Comment (k-jano) [inline src/components/NodeDetailCard.js:41]: Same :laughing:

File: src/components/NodeDetailCard.js (hunk 1 above)
Comment (filwie) [reply src/components/NodeDetailCard.js]: done

File: src/components/NodeDetailCard.js (hunk 2 above)
Comment (filwie) [reply src/components/NodeDetailCard.js]: done even harder!

**Comment by bzurkowski** [comment]
@filwie The feature looks great 👍 Tested locally, works nicely 😄 Together with node coloring and namespace filtering, the application finally became usable. Thanks!

To merge the change into the master branch, I allowed myself to rebase the change and open a new PR (#49).

**Pull request closed without merging**